husky_gazebo/launch/controller_utils.launch

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<launch>
+
+  <!-- Robot state publisher -->
+  <node pkg="robot_state_publisher" type="robot_state_publisher" name="robot_state_publisher">
+    <param name="publish_frequency" type="double" value="50.0" />
+    <param name="tf_prefix" type="string" value="" />
+  </node>
+
+ <!-- Joint state publisher -->
+  <node pkg="joint_state_publisher" type="joint_state_publisher" name="joint_state_publisher"/>
```

Comment:
AFAIK Joint state publisher tries to take responsibility for all non-fixed joints, which is bad, since something else should be publishing those states (ros_control)